docs/_documentations/troubleshooting.md

```diff
@@ -365,6 +365,30 @@ These steps reproduce the issue:
 
 **Workaround** Run the `Attach Debugger` action manually.
 
+<!--
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Codewind version: 0.5.0
+OS: Windows
+IDE extension version: 0.5.0
+IDE version: Eclipse 2019-09
+Issue link: https://github.com/eclipse/codewind/issues/715
+-->
+## Appsody projects stuck in starting state
+If you work on Appsody projects on Windows using the Eclipse IDE, you might see Appsody projects stuck in the `Starting` or `Stopped` state even though the container logs say the projects are up and running. This can happen when you create a number of projects using the default and Appsody templates with Codewind 0.5.0. 
+
+These steps reproduce the issue:
```

Comment:
Are the reproduction steps needed as part of the troubleshooting?  I think we should re-word this to let users know in what situations are they likely to run into this problem, e.g. after reinstalling Codewind and they open up to their workspace that has, many projects, etc.

Reply:
Makes sense, we do not need to list reproduction steps.

Reply:
Ok, have removed the reproduction steps.